docs/product-getting-started-guide/src/main/asciidoc/data-object-driver-create-proc.adoc

```diff
@@ -0,0 +1,84 @@
+[id='data-object-driver-proc']
+= Creating the Driver data object
+
+The following chapter instructs you how to create the Driver data object containing data fields, such as: Name, Age, and Total Points.
+
+//a traffic violation data model in {PRODUCT}. Data models are models of data objects. A data object is a custom complex data type (for example, a Violation data object containing data fields Violation Date, Fine Amount, and Speed Limit).
+
+[float]
+== Prerequisites
+
+You have created the Driver department traffic violations project.
+
+.Procedure
+. Log in to {CENTRAL}. Click *Menu* -> *Design* -> *Projects*, then *Driver department traffic violations*.
+. Click *Create New Asset* -> *Data Object*.
+. In the *Create new Data Object* wizard, enter the following values:
+.. *Data Object*: `Driver`.
+.. *Package*: select `org.jboss.example.traffic_violations`.
+. Click *+Ok*.
++
+
+.Create new Data Object window
+image::data-object2.png[]
+
+== Adding the Driver data object data fields
+
+. In the `'Driver'- general properties` section, locate the *Label* property and enter: *Driver*.
+. Click *+ add field*.
+. Enter the following values:
++
+* *Id*: `name`
+* *Label*: `Full Name`
+* *Type*: `String`
+
+. Click *Create and continue*, then enter the following values:
++
+* *Id*: `age`
+* *Label*: `Age`
+* *Type*: `Integer`
+
+. Click *Create and continue*, then enter the following values:
++
+* *Id*: `state`
+* *Label*: `State`
+* *Type*: `String`
+
+. Click *Create and continue*, then enter the following values:
++
+* *Id*: `city`
+* *Label*: `City`
+* *Type*: `String`
+
+. Click *Create and continue*, then enter the following values:
++
+* *Id*: `violations`
+* *Label*: `Violations`
+* *Type*: `Violation(org.jboss.example.traffic_violations.Violation)`
+* *List*: Mark this check box to enable the field to hold multiple items for the specified type.
```

Comment:
"Select" a check box, per IBM SG. I used to say "mark" check box as well, per Microsoft and Sun Microsystems style guides a previous gigs. I'll only mention it once, and I don't personally care. Just informing.

Reply:
done and noted for future ref